### DISCUSSION TITLE
]m and [m does not work

### DISCUSSION THREAD

**Discussion by sergii-trotsiuk** [JetBrains/ideavim#380] [Q&A]
I have tried use `]m` and `[m` and found that these motions do not work.

My IDEA:

IntelliJ IDEA 2021.2.2 (Community Edition)
Build #IC-212.5284.40, built on September 14, 2021
Runtime version: 11.0.12+7-b1504.28 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 5.11.0-34-generic
GC: ParNew, ConcurrentMarkSweep
Memory: 1896M
Cores: 8
Registry: debugger.watches.in.variables=false
Non-Bundled Plugins: AceJump (3.8.3), IdeaVIM (1.7.2), org.jetbrains.IdeaVim-EasyMotion (1.6), com.andrey4623.rainbowcsv (2.0), PythonCore (212.5284.44), idea.plugin.protoeditor (212.5080.8), org.exbin.deltahex.intellij (0.2.4), String Manipulation (8.19.203.000.0), MavenRunHelper (4.17.212.000.0), izhangzhihao.rainbow.brackets (6.21), com.haulmont.jpab (5.6-212)
Kotlin: 212-1.5.10-release-IJ5284.40
Current Desktop: ubuntu:GNOME

**Comment by AlexPl292** [comment]
Hi, what language have you tried?
Could you please also attach your `~/.ideavimrc`?

**Reply by sergii-trotsiuk** [reply]
Attached [ideavimrc.txt](https://github.com/JetBrains/ideavim/files/7419377/ideavimrc.txt) But it does not work even without .ideavimrc. Only `]M` and `[M` works.
I try it for Java.

**Reply by AlexPl292** [reply]
It seems like we have some issues with `]m` and `[m` implementation. Please observe this issue for the updates https://youtrack.jetbrains.com/issue/VIM-2447
